cluster/clientbin.sh

```diff
@@ -68,7 +68,7 @@ esac
 # Get the absolute path of the directory component of a file, i.e. the
 # absolute path of the dirname of $1.
 get_absolute_dirname() {
-  echo "$(cd "$(dirname "$1")" && pwd)"
+  cd "$(dirname "$1")" && pwd
```

Comment:
change reason:
```
In cluster/clientbin.sh line 71:
  echo "$(cd "$(dirname "$1")" && pwd)"
       ^-- SC2005: Useless echo? Instead of 'echo $(cmd)', just use 'cmd'.
```

Reply:
Not sure about this. Does running the cd in a subshell preserve $CWD?

Reply:
@liggitt 
Thanks a lot. As you said, there are some problems here, I will modify it later.